cmd/tigerblood/main.go

```diff
@@ -45,6 +45,16 @@ func startRuntimeCollector() {
 	c.Run()
 }
 
+func stringStringMapToStringUint(ss map[string]string) map[string]uint {
```

Comment:
I may be missing something, but why aren't the reputation weights stored as numbers in the config file?